src/c/PebbleAccelerometer.c

```diff
@@ -1,11 +1,63 @@
 #include <pebble.h>
 
 #define ACCEL_SAMPLE_RATE    ACCEL_SAMPLING_10HZ
+#define SAMPLES_PER_CALLBACK  1
+
+#define SYNC_BUFFER_SIZE      48
 
 static Window *s_main_window;
-static TextLayer *s_time_layer;
 static bool isCapturing = false;
 
+
+static TextLayer *s_time_layer;
+static int      syncChangeCount = 0;
+
+enum Pebble_Keys {
+  PP_KEY_CMD  = 128,
+  PP_KEY_X    = 1,
+  PP_KEY_Y    = 2,
+  PP_KEY_Z    = 3,
+};
+
+enum PebblePointer_Cmd_Values {
+  PP_CMD_INVALID = 0,
+  PP_CMD_VECTOR  = 1,
+};
+
+typedef struct {
```

Comment:
I will remove this. I was using it to test if AppSync works better than AppMesage. Never ended up using it but forgot to clear the code.